src/data/poolList.ts

```diff
@@ -234,122 +235,152 @@ export const poolList: Record<KnownNetwork, StaticPoolInfo[]> = {
                 symbol: '3L-BTC/USD',
-                decimals: TEST_TOKEN_DECIMALS,
+                decimals: TEST_QUOTE_TOKEN.decimals,
             },
             shortToken: {
                 name: '3-SHORT-BTC/USD',
-                address: '0x24f7dA47Db30F0Ef8f595A9AB0F5dE7c1b4E38Bc',
+                address: '0x16468a5c4Ba94C050a2E88B16A68b95D3CA0be2F',
                 symbol: '3S-BTC/USD',
-                decimals: TEST_TOKEN_DECIMALS,
-            },
-            quoteToken: {
-                name: 'USDC',
-                symbol: 'USDC',
-                address: '0x191f0Db87d3136541354F89F43a034E11287788D',
-                decimals: TEST_TOKEN_DECIMALS,
+                decimals: TEST_QUOTE_TOKEN.decimals,
             },
+            quoteToken: TEST_QUOTE_TOKEN,
         },
         {
             // 1x
             name: '1-ETH/USDC',
-            address: '0x34eE510e1d51904C50d9CD03199dE5147A4E15eD',
+            address: '0x6999d3f95b865cDE225a52d0157aA663CF2c78ef',
             leverage: 1,
             updateInterval: FIVE_MINUTES,
             frontRunningInterval: THIRTY_SECONDS,
-            keeper: '0xf3A03510d21c235C02203FF6725C9D1c0F4E49D9',
+            keeper: '0xFBCbB7184935122Fdf2c7e8b69a3207Ece722749',
             committer: {
-                address: '0x5F6a868e4AEa2AFCF56a7937c01896E564354542',
+                address: '0x9faE927FaF812450409Bc3259590d0D2F7add11C',
             },
             longToken: {
                 name: '1-LONG-ETH/USD',
-                address: '0x34966D8A3Ff227f13A5F8471FE6fa9D33a7f0B85',
+                address: '0xea7e9FdE89F6D8E07f83Fc722B64004A689fcBf6',
                 symbol: '1L-ETH/USD',
-                decimals: TEST_TOKEN_DECIMALS,
+                decimals: TEST_QUOTE_TOKEN.decimals,
             },
             shortToken: {
                 name: '1-SHORT-ETH/USD',
-                address: '0xf8e1F91966aEe3aD4CD83C9B09Cb947262aeB10A',
+                address: '0x5527FaA85146BE53cac459B61BA91a420705309c',
                 symbol: '1S-ETH/USD',
-                decimals: TEST_TOKEN_DECIMALS,
-            },
-            quoteToken: {
-                name: 'USDC',
-                symbol: 'USDC',
-                address: '0x191f0Db87d3136541354F89F43a034E11287788D',
-                decimals: TEST_TOKEN_DECIMALS,
+                decimals: TEST_QUOTE_TOKEN.decimals,
             },
+            quoteToken: TEST_QUOTE_TOKEN
         },
         {
             // 3x
             name: '3-ETH/USDC',
-            address: '0xe3cAD84c29775399a25c6Ff040FE141b4eD10047',
+            address: '0x11F7FD07422dB02AceE8009C002146a3b0980D92',
+            leverage: 3,
+            updateInterval: FIVE_MINUTES,
+            frontRunningInterval: THIRTY_SECONDS,
+            keeper: '0xFBCbB7184935122Fdf2c7e8b69a3207Ece722749',
+            committer: {
+                address: '0x0212e821a5383dB36362f197fdb0Dc93A75508C4',
```

Comment:
this should be `0x29EAAcB6FF4B883F1Ed630425cE89932601753E1` my bad, have updated notion doc to match (using graph as source of truth)